src/ResourceManager/ApiManagement/Commands.ApiManagement.ServiceManagement/packages.config

```diff
@@ -1,9 +1,12 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="AutoMapper" version="6.0.2" targetFramework="net452" />
-  <package id="Microsoft.Azure.Management.ApiManagement" version="4.0.2-preview" targetFramework="net452" />
+  <package id="Microsoft.Azure.Management.ApiManagement" version="4.0.3-preview" targetFramework="net452" />
   <package id="Microsoft.Data.Edm" version="5.8.2" targetFramework="net452" />
   <package id="Microsoft.Data.OData" version="5.8.2" targetFramework="net452" />
+  <package id="Microsoft.Rest.ClientRuntime" version="2.3.11" targetFramework="net452" />
```

Comment:
Remove Microsoft.Rest.ClientRuntime and Microsoft.Rest.ClientRuntime.Azure, as these are imported by common dependencies.